### DISCUSSION TITLE
截图黑屏问题

### DISCUSSION THREAD

**Discussion by xiaobai-mc** [hiroi-sora/Umi-OCR#515] [Q&A]
截图的时候，那个十字鼠标太夸张了。然后屏幕会黑一下。一张还好，连续十来张眼睛有点受不了，想问下有办法解决

**Comment by hiroi-sora** [comment]
截屏时黑屏闪烁，是显卡驱动的兼容性问题。

解决方法：

全局设置 → 渲染器 → 改为`禁用硬件加速`或者`Desktop OpenGL`

![image](https://github.com/hiroi-sora/Umi-OCR/assets/56373419/097535ea-503b-4267-813f-4775f655193c)

**Reply by xiaobai-mc** [reply]
不对啊。我电脑本来只是屏幕变暗一点点，关了或换成另外一个更是黑屏闪烁。。
因为我这边偶尔需要大量的屏幕截图。这个用一会眼睛真累。

**Reply by hiroi-sora** [reply]
哦哦，就说是原本并不会闪烁，但你不喜欢截图时屏幕变暗，对吗？

那么，请先将渲染器恢复原来的ES那个设置。

然后，用记事本打开`UmiOCR-data/qt_res/qml/ImageManager/ScreenshotWindowComp.qml`这个文件。前面的一行代码指定了截图时深色叠加层的颜色：
```
property color darkLayerColor: "#73000000"
```

你可以将前两个数字改得小一点，这样颜色会更浅，比如 "#12000000" 。

改完后，记得保存文件、重启软件。你可以尝试改到一个自己认为舒服的数值。

**Reply by xiaobai-mc** [reply]
对对。。我没表达清楚。谢谢。我试试看

**Reply by hiroi-sora** [reply]
嗯，然后如果你不喜欢大十字光标，可以将 crossLineColor 十字指示器的颜色 改为透明色（`#00000000`，8个0）。

**Comment by xiaobai-mc** [comment]
大佬，晚上好。我有台电脑报错 PaddleOCR-json.exe （0xc000001d）&nbsp;无法正常启动，是什么情况啊！！！

**Reply by hiroi-sora** [reply]
你的cpu可能不含AVX指令集。 PaddleOCR 插件依赖AVX指令集来调用mkldnn数学加速，没有此指令集就会报错。

建议你重新下载使用 RapidOCR 引擎版本的 Umi-OCR 。 Rapid 的硬件兼容性更好，不依赖avx指令集。。

**Comment by xiaobai-mc** [comment]
好的，装了半天运行库。解决不了，你们那个网站一会又打不开。明天试试看。qqq




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "hiroi-sora/Umi-OCR"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月25日(星期五) 晚上11:45
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [hiroi-sora/Umi-OCR] 截图黑屏问题 (Discussion #515)





 
你的cpu可能不含AVX指令集。 PaddleOCR 插件依赖AVX指令集来调用mkldnn数学加速，没有此指令集就会报错。
 
建议你重新下载使用 RapidOCR 引擎版本的 Umi-OCR 。 Rapid 的硬件兼容性更好，不依赖avx指令集。。

**Comment by xiaobai-mc** [comment]
大大你好，请问下 PaddleOCR有翻译功能没？
